### ISSUE TITLE
ignore-non-standard does not ignore -webkit-tap-highlight-color

### ISSUE THREAD

**Issue by PinPinIre** [sasstools/sass-lint#843]
What version of Sass Lint are you using?
1.9.0

Please include any relevant parts of your configuration

``` YAML
rules:
  no-vendor-prefixes:
    - 2
    -
      ignore-non-standard: true
```

``` SCSS
html {
  -webkit-tap-highlight-color: $link-color-hover;
}

button::-moz-focus-inner,
input::-moz-focus-inner {
  border: 0;
  padding: 0;
}

input[type="number"]::-webkit-inner-spin-button,
input[type="number"]::-webkit-outer-spin-button {
  height: auto;
}
```

What did you do? Please include the actual source code causing the issue.
I ran the no-vendor-prefixes rule on the example for `ignore-non-standard` which then caused an error. If the `ignore-non-standard` config is set to false, 5 errors are thrown which is expected.

What did you expect to happen?
The -webkit-tap-highlight-color property to be ignored when `ignore-non-standard: true`.

What actually happened? Please include any error messages given to you by Sass Lint.
An error was thrown.
The -webkit-tap-highlight-color property caused an error to be thrown.
`2:3    Error   Vendor prefixes should not be used  no-vendor-prefixes`

If you're using a IDE plugin have you tried the CLI too?
The above occurred using the CLI.

**Comment by chrismbarr** [comment]
I've noticed the same thing on `1.9.1` as well.  It's actually more than just this one property.  I get lint warnings for all of these as well:

```scss
-ms-overflow-style: scrollbar;
-ms-touch-action: manipulation;
-webkit-font-smoothing: antialiased;
-webkit-tap-highlight-color: transparent;
```

**Comment by DanPurdy** [comment]
I think this is to do with what's happened to our property list. It's been used in the correct property spelling rule which means every property has been dumped in it including experimental ones. What I think is gonna have to happen is to split it into two separate lists, one with all known properties added and one with only standardised properties in, only way I can see to rectify this properly.

**Comment by DanPurdy** [comment]
Sorry about the wait, had to take a bit of an unschedule hiatus for a while there.

We will be releasing a V2 of sass-lint in the next couple of weeks to mainly drop support for old node versions but also to replace the vendor prefix rule. I've moved rules similar to stylelint's approach to this over to sass-lint but with added customisation options! Hopefully should remove the issues you've all been having.

It goes off of the autoprefixer / caniuse definitions but also allows you to specify properties / values / elements etc that you would like to override.
